src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/config/AwsLocalStackConfiguration.kt

```diff
@@ -33,14 +34,24 @@ class AwsLocalStackConfiguration(
       .build()
   }
 
+  @Bean
+  fun calculationCompleteEventsAmazonSQSAsync(
+    @Value("\${calculation-complete.sqs-endpoint}") serviceEndpoint: String
+  ): AmazonSQSAsync {
+    return AmazonSQSAsyncClientBuilder.standard()
+      .withEndpointConfiguration(AwsClientBuilder.EndpointConfiguration(serviceEndpoint, region))
+      .withCredentials(AWSStaticCredentialsProvider(AnonymousAWSCredentials()))
```

Comment:
Why does this one work? is it pub/sub issue or an sqs/sns issue?

Reply:
I think we publish to both SQS and SNS in these tests - SNS publish on localstack I think is what caused the error. Other applications have no credentials set here at all e.g. 

https://github.com/ministryofjustice/court-register/blob/76cc4b748d9faf3b6a610d1c895962d863cb7bf7/src/main/kotlin/uk/gov/justice/digital/hmpps/courtregister/config/LocalstackConfig.kt

I couldn't work out where the credentials come from there, maybe environment variables

 I can't find a reference for the problem now though